pytket/docs/circuit.rst

```diff
@@ -14,104 +14,104 @@ pytket.circuit
 
 .. automodule:: pytket._tket.circuit
     :members: fresh_symbol
-.. autoclass:: pytket._tket.circuit.Op
```

Comment:
I think all of this _tket stuff is replaced in the docs build anyway. I don't think theres any good reason for it to be in the rst source